.github/workflows/maven.yml

```diff
@@ -29,7 +29,7 @@ on:
 jobs:
   build:
 
-    runs-on: macos-latest
+    runs-on: ubuntu-latest
```

Comment:
I'm not sure about this. Shardingsphere uses macos.

Reply:
I think Linux is more common in actual environment and it is a little faster than macOS.